api/server.go

```diff
@@ -34,9 +35,26 @@ func (d *dispatcher) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // Server is a Docker API server.
 type Server struct {
-	hosts      []string
-	tlsConfig  *tls.Config
-	dispatcher *dispatcher
+	cfg          *Config
+	hosts        []string
+	tlsConfig    *tls.Config
+	dispatcher   *dispatcher
+	authZPlugins []authorization.Plugin
+}
+
+// versionMatcher defines a variable matcher to be parsed by the router
+// when a request is about to be served.
+const versionMatcher = "/v{version:[0-9.]+}"
```

Comment:
Good idea to make this a constant, but I don't see it used. Looks like it should be used in `api/primary.go`.